config/config_params.go

```diff
@@ -205,6 +205,8 @@ type Config struct {
 	sourceToRawConfig map[Source]map[string]string
 	rawValues         map[string]string
 	Err               error
+
+	IptablesNATOutgoingInterfaceFilter string `config:"string;"`
```

Comment:
We should validate this, I think.  To add a validator:
- change `string` to, say `iface-pattern`
- add a new `case "iface-pattern"` to the `switch` in the `loadParams()` function:
- in that `case`, you should be able to use a RegexpPattern like the one for `iface-list`.  However, you'll need a different regexp.  I think this regexp should work: `^[a-zA-Z0-9:._+-]{1,15}$` i.e. 1-15 characters consisting of alphanumerics, colon, dot, underscore, hyphen and plus.

Reply:
ack